examples/golang/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+FROM apache/skywalking-eyes:latest
```

Comment:
How about we push `version:latest-go-deps` image as well to make it convenient?

Reply:
> How about we push `version:latest-go-deps` image as well to make it convenient?

I'm not sure whether it'll be convenient, people might be using different go version and it does not make things much better unless we want to push tens of tags for each version